js/src/get-global-config/index.ts

```diff
@@ -90,6 +91,7 @@ export const main_ = (config: lib.Config, input: Input): Result => {
     enable_tfsec: config?.tfsec?.enabled ?? false,
     enable_tflint: config?.tflint?.enabled ?? true,
     enable_trivy: config?.trivy?.enabled ?? true,
+    tflint_fix: config?.tflint?.fix ?? false,
```

Comment:
Add the output `tflint_fix` to get-global-config and js/get-global-config.
The default value is false.